### ISSUE TITLE
Please increase indexers to 20

### ISSUE THREAD

**Issue by Corrupta** [theotherp/nzbhydra#41]
Please increase indexers to 20

**Comment by theotherp** [comment]
What the fuck...

**Comment by Corrupta** [comment]
lol just really need 1 more extra but would be nice to go 20 I mean why the fuck not you never know

**Comment by Corrupta** [comment]
lol think that will be a while, thanks

**Comment by albino1** [comment]
I have 27 indexers... just sayin' :)

In all seriousness though, less than 10 of them are worth using, and 99% of my files come from like 3 of them.  I only kept them all in NZBM in the off chance one of them magically became good.  Turns out that never happens.  The odds of one of the bottom 20 indexers having something that the top 3 indexers don't have is essentially zero -- unless it's spam or malware, in which case I obviously don't want it anyway.
